lib/Models/CsvCatalogItem.ts

```diff
@@ -140,6 +141,82 @@ export default class CsvCatalogItem extends TableMixin(
       );
     }
   }
+
+  @computed
+  get selectableDimensions() {
```

Comment:
This adds dimensions for changing region mapping for YDYR (selecting region column and region type).
It would be nice to hide this by default as most people won't want to see this. I was thinking of maybe adding an `advancedDimension` boolean and adding an option in the workbench item menu to "Show advanced options" or something.

![csv-dimensions](https://user-images.githubusercontent.com/6187649/79307700-a9b0fa00-7f3a-11ea-99a5-ecc4c3242e07.png)